### ISSUE TITLE
Train slowly on MLT dataset

### ISSUE THREAD

**Issue by gitUserGoodLeaner** [insightcs/PAN-Pytorch#1]
@insightcs Very nice work for curve text detection, I had training on MLT task by this repository code, however the training seems very slow. Could you share your training settings? Like batch size, learning rate, or other training related configs. Thanks  in advance!
